encrypted-media-respec.html

```diff
@@ -3825,25 +3825,25 @@
 
           <section id="per-origin-individualization">
             <h5>Per-Origin Individualization</h5>
-            <p>Per-<a def-id="origin"></a> individualization is performed between the <a def-id="cdm"></a> and an application-selected server and results in a per-origin identifier.
-              The process MUST be performed via these APIs and MUST NOT involve other types of communication.
-              Any <a def-id="distinctive-identifiers"></a> used MUST be origin-specific.
+            <p>Per-<a def-id="origin"></a> individualization is performed between the <a def-id="cdm"></a> and the application, including an application-selected server, and results in a per-origin identifier.
+              The process MUST be performed via these APIs and MUST NOT involve other methods of communication.
+              As with all other uses of the APIs, <a def-id="distinctive-identifiers"></a> MAY be used but indelible or non-origin-specific MUST NOT be exposed, even in encrypted form.
+              If one or more <a def-id="distinctive-identifiers"></a> is used, the resulting identifier is by definition also a <a def-id="distinctive-identifier"></a>.
```

Comment:
Is this strictly true ? The approach of hashing one identifier with a smaller random salt can create a new identifier from a space that is large enough that it is not associable but small enough that it has useful non-migratability properties.